microsoft-365/enterprise/move-sharepoint-between-geo-locations.md

```diff
@@ -224,4 +224,6 @@ Power Apps need to be recreated in the destination location.
 
 ### Data movement between geo locations
 
-SharePoint uses Azure Blob storage for its content, while the metadata associated with sites and its files is stored within SharePoint. After the site is moved from its source geo location to its destination geo location, the service will also move its associated Blob Storage. Blob Storage moves complete in approximately 40 days.
+SharePoint uses Azure Blob storage for its content, while the metadata associated with sites and its files is stored within SharePoint. After the site is moved from its source geo location to its destination geo location, the service will also move its associated Blob Storage. Blob Storage moves complete in approximately 40 days, this will not have any impact to users interation with the data. 
```

Comment:
```suggestion
SharePoint uses Azure Blob Storage for its content, while the metadata associated with sites and its files is stored within SharePoint. After the site is moved from its source geo location to its destination geo location, the service will also move its associated Blob Storage. Blob Storage moves complete in approximately 40 days, this will not have any impact to users interation with the data. 
```